.travis.yml

```diff
@@ -22,7 +22,10 @@ script:
   - bundle exec yard-junk
 
 install:
-  - gem install bundler
+  - if [ $TRAVIS_RUBY_VERSION == '2.2' ] || [ $TRAVIS_RUBY_VERSION == '2.1' ] || [ $TRAVIS_RUBY_VERSION == '2.0' ];
```

Comment:
Following the [daru-view travis file](https://github.com/SciRuby/daru-view/blob/master/.travis.yml#L20)